common/src/main/scala/package.scala

```diff
@@ -59,6 +60,9 @@ package object eoi {
       }
     }
 
+    def formatPercentage: String = NF.getPercentInstance().format(in)
+    def formatSterling: String = "£" + NF.getInstance().format(in)
```

Comment:
wouldn't you use `getCurrencyInstance` instead of `getInstance` here?

Reply:
Good suggestion. Done.